packages/react/src/hooks/constants.ts

```diff
@@ -0,0 +1,272 @@
+import {
+  SendUserAttributeVerificationCodeOutput,
+  UpdateUserAttributesOutput,
+  UserAttributeKey,
+} from "aws-amplify/auth";
+
+export type UserAttributes = Record<UserAttributeKey, string | undefined>;
+
+export const DefaultAttributes: UserAttributes = {
+  email: undefined,
+  phone_number: undefined,
+  address: undefined,
+  birthdate: undefined,
+  email_verified: undefined,
+  family_name: undefined,
+  gender: undefined,
+  given_name: undefined,
+  locale: undefined,
+  middle_name: undefined,
+  name: undefined,
+  nickname: undefined,
+  phone_number_verified: undefined,
+  picture: undefined,
+  preferred_username: undefined,
+  profile: undefined,
+  sub: undefined,
+  updated_at: undefined,
+  website: undefined,
+  zoneinfo: undefined,
+};
+
+// Default object to be passed into useDataState
+export const defaultUpdateUserAttributesOutput: UpdateUserAttributesOutput = {
```

Comment:
Does the output of `updateUserAttributes` always include every field?

Reply:
No, it only includes the fields you submit. However, when feeding it into useDataState, I see an error unless all the standard attributes are included. Likely some typing issue, using a `Partial<UpdateUserAttributesOutput>` does not resolve it.